dev/README.md

```diff
@@ -2,7 +2,7 @@
 
 
 ## Debugging and profiling (`dev/`)
-**More info on debugging: [`doc/debugging.md`](doc/debugging.md)**
+**More info on debugging: [`dev/doc/debugging.md`](doc/debugging.md)**
```

Comment:
Why this change?

Reply:
For consistency with the rest of the document. I noticed this while re-reading it.